<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilmScannerWindow</class>
 <widget class="QDialog" name="FilmScannerWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Film scanner</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/window/film-scanner</normaloff>:/window/film-scanner</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,1,0,0,0,0,0">
   <item>
    <layout class="QHBoxLayout" name="layoutDirectory">
     <item>
      <widget class="QLabel" name="labelDirectory">
       <property name="text">
        <string>Directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="eDirectory">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bSelectDirectory">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cSearchInSubdirs">
     <property name="text">
      <string>Search in subdirectories</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutScan" stretch="1,2,1">
     <item>
      <spacer name="spacerScanLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bScan">
       <property name="text">
        <string>&amp;Scan</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacerScanRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="FoundedListWidget" name="gbFounded" native="true"/>
   </item>
   <item>
    <widget class="QCheckBox" name="cSearchForPoster">
     <property name="text">
      <string>Search for a poster with the same filename in the directory</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cLoadInformation">
     <property name="text">
      <string>Load information from online source</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wLoadInfoSettings" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="layoutLoadInfoSettingsMain">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layoutLoadInfoSettings">
        <item>
         <layout class="QHBoxLayout" name="layoutSource">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QComboBox" name="cbSource"/>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="cDownloadBigPoster">
          <property name="text">
           <string>Download big poster (if possible)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cDownloadMoreInformation">
          <property name="text">
           <string>Download more information (if possible)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="lineMain0">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutFooter" stretch="0,0,0,0,0,1,0,0,0">
     <item>
      <widget class="QLabel" name="labelTotalFounded">
       <property name="text">
        <string>Total founded:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lTotalFounded">
       <property name="text">
        <string notr="true">0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="lineFooter0">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSelected">
       <property name="text">
        <string>Selected:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lSelected">
       <property name="text">
        <string notr="true">0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacerFooter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bAdd">
       <property name="text">
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FoundedListWidget</class>
   <extends>QWidget</extends>
   <header>scanner/foundedlistwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bClose</sender>
   <signal>clicked()</signal>
   <receiver>FilmScannerWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cLoadInformation</sender>
   <signal>toggled(bool)</signal>
   <receiver>wLoadInfoSettings</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
